test/controllers/ApplicationSpec.scala

```diff
@@ -108,9 +127,10 @@ class ApplicationSpec  extends WordSpecLike with Matchers with EitherValues {
                                      |  "after": "335aa74f382a36c7a3594a8fec14fdd2ac754cb2"
                                      |}""".stripMargin))
 
-      status(call(application.update(), request2)) shouldBe OK
+      contentAsString(call(application.update(), request3)) shouldBe "Site update requested"
```

Comment:
I could do that as well - I'm not sure it is necessary though - you will only get that text given an OK response. I felt that it was more useful to check the text here given that a unauthorised attempt will yield response text that is useful to update the tests with i.e. the SHA1.